<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>      Number</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Power Ball</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="number_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PMingLiU-ExtB</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#FFFFFF;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="number_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PMingLiU-ExtB</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#FFFFFF;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="number_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PMingLiU-ExtB</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#FFFFFF;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="number_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PMingLiU-ExtB</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#FFFFFF;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="number_5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PMingLiU-ExtB</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#FFFFFF;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PowerBall">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>130</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PMingLiU-ExtB</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ff0000;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_create">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>370</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:bold;</string>
    </property>
    <property name="text">
     <string>Create Power Ball </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>837</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
